configure.sh

```diff
@@ -5,7 +5,6 @@ CIRCT_PATH=
 TARGET="release"
 LLVM_CONFIG_BIN="llvm-config-16"
 ENABLE_COVERAGE="no"
-CIRCT_ENABLED="no"
```

Comment:
Why did you remove this one? It should rather stay and be renamed `ENABLE_HLS` as that is what it does, no?

Reply:
Because in the root Makefile there's an "ifdef CIRCT_ENABLED" to decide if to include it in the compilation or not.
With setting it to "no" the variable gets defined and always included in the compilation.

Reply:
It's now HLS_ENABLED